modules/ROOT/pages/reference/procedures.adoc

```diff
@@ -907,6 +939,58 @@ m|READ
 // m|reader, editor, publisher, architect, admin
 |===
 
+[[procedure_db_index_vector_createNodeIndex]]
+.db.index.vector.createNodeIndex()  label:new[Introduced in 5.11] label:beta[]
+[cols="<15s,<85"]
+|===
+| Description
+a|
+Create a named vector index for the given label, and property with given vector dimensionality, using the given similarity function.
+
+Valid case-insensitive values for the `vectorSimilarityFunction` are:
+
+* `"euclidean"` -- Euclidean similarity, where:
++
+[latexmath]
+++++
+\mathrm{Euclidean}(\vec v,\vec u)
+    := \dfrac1{1 + \left\lVert\vec v - \vec u\right\rVert^2}
+++++
++
+
+* `"cosine"` -- Cosine similarity, where:
++
+[latexmath]
+++++
+\begin{align}
+    \mathrm{cosign}(\vec v,\vec u)
```

Comment:
Typo in `cosine`, and I would probably clarify / make explicit the unit vector notation :)

Reply:
ah bugger, I thought I got all of those.
I'll fix that in the SVG render I need to make.

Well the unit vector notation is clarified on the next line of maths :wink: 
However, I don't think it needs too much more clarification as those that are likely double-checking the mathematical calculation of a score from a vector-based thing … likely know the basics of vectors.

Reply:
> likely know the basics of vectors

Well, like most mathematical notation it's less standard / ubiquitous than one would've liked, so I wouldn't count on it :P

Reply:
Thoughts on that for context? It's the same in both.

Reply:
Perfect :+1: